otx/algorithms/detection/adapters/openvino/task.py

```diff
@@ -527,7 +543,7 @@ def add_prediction(id: int, predicted_scene: AnnotationSceneEntity, aux_data: tu
 
             if add_saliency_map and saliency_map is not None:
                 labels = self.task_environment.get_labels().copy()
-                if saliency_map.shape[0] == len(labels) + 1:
+                if len(saliency_map) == len(labels) + 1:
```

Comment:
Can I ask one more question to check if I understood correctly? 
Does it mean that before saliency map was an array with `(num_classes, feat_w, feat_h)` shape, 
and now it's a list of `num_classes` np.arrays with `(feat_w, feat_h)` size or `None`, if this class wasn't detected?

Reply:
Yeah, that is correct. 
For now, these data strictures can handle all the cases that we have.

But if/when we will support per-box as well, we will need to update it probably.